libafl/src/executors/command.rs

```diff
@@ -828,6 +820,18 @@ pub trait CommandConfigurator<I, C = Child>: Sized {
     /// Set the timeout duration for execution of the child process.
     fn exec_timeout_mut(&mut self) -> &mut Duration;
 
+    /// Maps the exit status of the child process to an `ExitKind`.
+    #[inline]
+    fn exit_kind_from_status(&self, status: &std::process::ExitStatus) -> ExitKind {
+        use crate::std::os::unix::process::ExitStatusExt;
```

Comment:
can't you just delete this?

Reply:
Not really. It is the trait providing `signal` method for UNIX environments.